librz/core/cmd_analysis.c

```diff
@@ -11,7 +11,7 @@ static const char *help_msg_a[] = {
 	"a*", "", "same as afl*;ah*;ax*",
 	"aa", "[?]", "analyze all (fcns + bbs) (aa0 to avoid sub renaming)",
 	"a8", " [hexpairs]", "analyze bytes",
-	"ab", "[b] [addr]", "analyze block at given address",
+	"ab", "[?] [b] [addr]", "analyze block at given address",
 	"abb", " [len]", "analyze N basic blocks in [len] (section.size by default)",
```

Comment:
i believe this is already in the `ab` sub-menu. I'd say to remove this and just keep a nice description under `ab?`.

Reply:
Is edb0e63c3c910e8f9633508068a31ab4e7977b92 what you meant over here?
`ab` already had a proper description inside `ab?`

Reply:
No i meant that if you look at `help_msg_ab` there is already `abb`, so I see no point in having `abb` listed also here. I think we can remove this and just keep the help under `help_msg_ab`.

Reply:
Oh yeah, didn't notice that, sorry.
Removed that.